lib/rust/ensogl/pack/js/src/runner/config.ts

```diff
@@ -335,6 +335,7 @@ export class Group<Options extends OptionsRecord, Groups extends GroupsRecord> {
         }
         entries.sort()
         return entries
+            // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
`_` in general should be ignored by typescript but here is not. Suppressing is more readable than complicating the function.

Alternatively we could overwrite `varsIgnorePattern` in the config as suggested [here](https://stackoverflow.com/questions/64052318/how-to-disable-warn-about-some-unused-params-but-keep-typescript-eslint-no-un) but for now we struggle only here lets not overengineer things